.pre-commit-config.yaml

```diff
@@ -12,3 +12,15 @@ repos:
     hooks:
     -   id: autopep8
         args: ["-i"]
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-added-large-files
+        exclude: docs/data
```

Comment:
?

We don't want any large files in there. Hopefully none got through with all the plotting example PRs...

It does look like you might have added one here: `docs/data/pbmc3k_raw.h5ad`

Reply:
Yeah, true.
Think this one origins from me building the docs. I actually thought that this was already there, but apparently not. Hence, I excluded this path.

I will remove the file & remove the exclude.